modules/system/controllers/updates/assets/src/components/Product.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+    <div class="product-card p-2 mb-1">
+        <div class="product-body">
+            <div class="product-row relative">
+                <div class="product-image">
+                    <img :src="product.icon" :alt="product.name">
+                </div>
+                <div class="product-description">
+                    <div>
+                        <p class="product-name">{{product.name}}</p>
+                        <p>{{product.description}}</p>
+                    </div>
+                </div>
+                <div class="absolute">
+                    <button v-if="!product.installed"
+                            class="btn btn-info"
+                            data-control="popup"
+                            data-handler="onInstallPlugin"
+                            :data-request-data="`package: '${product.package}'`"
+                    >Install</button>
+                    <p v-if="product.installed" class="text-muted">This {{type}} is installed.</p>
+                </div>
+            </div>
+        </div>
+        <div class="product-footer">
+            <div class="product-footer-item">
+                <div title="Stars given" class="stars">
+                    <span class="product-badge"><i class="icon-star"></i></span>
+                    {{product.favers}}
+                </div>
+                <div title="Downloads" class="downloads">
+                    <span class="product-badge"><i class="icon-download"></i></span>
+                    {{product.downloads}}
+                </div>
```

Comment:
We should probably add the license as well

Reply:
Sure, it'll have to come from the marketplace tho as the packagist api does not provide it